eth/contracts/PancakeStableSwap/PancakeStableSwapTwoPool.sol

```diff
@@ -0,0 +1,774 @@
+// SPDX-License-Identifier: MIT
```

Comment:
Please add tests as well :pray:  https://github.com/pancakeswap/pancake-smart-contracts/blob/master/projects/stable-swap/test/PancakeStableSwapTwoPool.test.ts It can be in other PR (tests included in CI)

Reply:
discussed offline, under nice to have task